tests/test_history_client.py

```diff
@@ -56,7 +56,7 @@ async def test_history_aggregate(
     history_client: HistoryClient, mocker: MockerFixture
 ) -> None:
     TIME = Timestamp(0)
-    AGGREGATE = create_autospec(history_pb2.HistoryResponse.Aggregate, set_spec=True)
+    AGGREGATE = create_autospec(history_pb2.HistoryResponse.Aggregate, setspec=True)
```

Comment:
```suggestion
    AGGREGATE = create_autospec(history_pb2.HistoryResponse.Aggregate, spec_set=True)
```

Why does it even work!?!?!

Reply:
If you use `set_spec`, you'll get raised an exception with:
`RuntimeError: 'set_spec' might be a typo; use unsafe=True if this is intended`

As that disappeared with removing the underscore, I assumed it's fixed. But don't ask me why the test passes with and without the parameter :(